.github/workflows/build-images-base.yaml

```diff
@@ -23,10 +23,6 @@ on:
         description: DNS Operator bundle version
         default: latest
         type: string
-      wasmShimVersion:
```

Comment:
heads up @didierofrivia 

The wasm shim is no longer part of the operator bundle. Instead, it is added as part of the kuadrant operator image build process.